src/monitor/monitor.c

```diff
@@ -72,11 +72,6 @@
  */
 #define KRB5_RCACHE_DIR_DISABLE "__LIBKRB5_DEFAULTS__"
 
-/* Warning messages */
-#define CONF_FILE_PERM_ERROR_MSG "Cannot read config file %s. Please check "\
-                                 "that the file is accessible only by the "\
-                                 "owner and owned by root.root.\n"
```

Comment:
Hm... https://github.com/SSSD/sssd/pull/6841 should change this wording...

Anyway, lets have different messages in 'monitor' and 'sssd_kcm' for a moment being.

Reply:
> Hm... https://github.com/SSSD/sssd/pull/6841 should change this wording...

Are you referencing the right PR? That PR is already pushed (so its changes should already be included here) and it doesn't affect this message.

> Anyway, lets have different messages in 'monitor' and 'sssd_kcm' for a moment being.

What do you mean? Having the same sentence duplicated on both files or also changing the text?

Reply:
> Are you referencing the right PR?

I mean I missed to update this message in that PR.

> What do you mean? Having the same sentence duplicated on both files or also changing the text?

This part will be completely re-done in #6908
If you introduce a shared define now, it will complicate things.

Reply:
OK. So duplicated the string so PR #6908 can just remove the one from monitor.c